gke/standard/regional/single-zone/main.tf

```diff
@@ -0,0 +1,28 @@
+/**
+* Copyright 2024 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START gke_standard_regional_single_zone]
+resource "google_container_cluster" "default" {
+  name               = "gke-standard-zonal"
```

Comment:
this creates a regional cluster, the name "gke-standard-zonal" using 'zonal' is misleading, IMO